pkg/openshift/rosa/accountroles.go

```diff
@@ -0,0 +1,154 @@
+package rosa
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/openshift/osde2e-common/internal/cmd"
+)
+
+// accountRoles represents all roles for a given prefix/version
+type accountRoles struct {
+	controlPlaneRoleARN string
+	installerRoleARN    string
+	supportRoleARN      string
+	workerRoleARN       string
+}
+
+// accountRolesError represents the custom error
+type accountRolesError struct {
+	action string
+	err    error
+}
+
+// Error returns the formatted error message when accountRolesError is invoked
+func (a *accountRolesError) Error() string {
+	return fmt.Sprintf("%s account roles failed: %v", a.action, a.err)
+}
+
+// createAccountRoles creates the account roles to be used when creating rosa clusters
+func (r *Provider) createAccountRoles(ctx context.Context, prefix, version, channelGroup string) (*accountRoles, error) {
+	const action = "create"
+	var (
+		accountRoles *accountRoles
+		err          error
+	)
+
+	r.log.Info("Checking whether account roles exist with", prefixLoggerKey, prefix, versionLoggerKey, version)
+	if accountRoles, err = r.getAccountRoles(ctx, prefix, version); err != nil {
+		return nil, &accountRolesError{action: action, err: err}
+	}
+
+	if accountRoles == nil {
+		r.log.Info("Creating account roles with", prefixLoggerKey, prefix, versionLoggerKey, version)
+
+		commandArgs := []string{
+			"create",
+			"account-roles",
+			"--prefix",
+			prefix,
+			"--version",
+			version,
+			"--channel-group",
+			channelGroup,
+			"--mode",
+			"auto",
+			"--yes",
+		}
+
+		// TODO: Open an RFE to rosa to support --output option
+		if _, stderr, err := r.RunCommand(ctx, exec.CommandContext(ctx, r.rosaBinary, commandArgs...)); err != nil {
```

Comment:
```go
// RunCommand runs the rosa command provided
func (r *Provider) RunCommand(ctx context.Context, command *exec.Cmd) (io.Writer, io.Writer, error) {
	command.Env = append(command.Environ(), r.awsCredentials.CredentialsAsList()...)
>>>>	r.log.Info("Command", rosaCommandLoggerKey, strings.Split(command.String(), "bin/")[1])
	return cmd.Run(command)
}
```

This will be done for every rosa command issued as each calls the `RunCommand` method which will print the command.